doc/architecture/decisions/0002-object-identifiers.md

```diff
@@ -0,0 +1,57 @@
+# 0002. Object identifiers
+
+Date: 2019-06-11
+
+## Status
+
+Agreed
+
+## Context
+
+To uniquely identify an object in Ensembl we need three pieces of information:
+
+-   The **dataset** to which the object belongs, e.g. homo_sapiens_GCA_000001405_14 [^1]
+-   The **object type**, e.g. gene, transcript, region  
+-   The **object ID**, e.g. ENSG00000139618, AT3G52430, 13:32315474-32400266 [^2]  
+  
+Ensembl objects are used extensively in the web client and passing around three separate pieces of information for each one can be cumbersome, error prone, and adds additional work when serialisation is required (e.g. for use in URLs or hash keys).  
+
+[^1]: For our purposes dataset can be considered synonymous to genome_id
```

Comment:
Helps if I read further down. I would try to remove that ambiguity in the prose.